src/Packagist/Api/Result/Factory.php

```diff
@@ -38,6 +38,10 @@ public function create(array $data)
                 // Used for /explore/popular.json
                 return $this->createSearchResults($data['packages']);
             }
+            if (isset($data['minified']) && 'composer/2.0' === $data['minified']) {
```

Comment:
Not using `composer/metadata-minifier` to unminify the metadata of versions looks suspicious to me (and might be the reason why you needed to patch `createPackageResults` when using the metadata of each version).

Reply:
Hey @stof, would you be interested in submitting a patch to implement that package instead of this method?